test/TestProjects/MgmtMockTest/Samples/Generated/DiskEncryptionSets_CreateOrUpdate$$Create a disk encryption set with key vault from a different subscription..cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#region Usings
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Resources;
+using MgmtMockTest;
+using MgmtMockTest.Models;
+#endregion
```

Comment:
Just out of curiosity, why do we need these regions? Are these related with something in the api explorer?